xml/troubleshooting.xml

```diff
@@ -729,59 +726,12 @@
         </listitem>
       </itemizedlist>
       <para>
-        In all cases that do not involve external network problems, the
-        solution is to reboot the system into single-user mode and repair the
-        configuration before booting again into operating mode and attempting
-        to log in again. To boot into single-user mode:
+        In all cases that do not involve external network problems, the solution
+        is to reboot the system into the rescue mode and repair the
+        configuration before booting again into operating mode and attempting to
+        log in again. To boot into the rescue targe, follow the procedure
+        outlined in <xref linkend="proc-trouble-login-no"/>.
```

Comment:
This would need to change if you change the xml:id of the procedure in grub2.xml
```suggestion
        outlined in <xref linkend="proc-rescue-mode"/>.
```